detox/src/ios/expect.js

```diff
@@ -65,9 +65,9 @@ class TapAtPointAction extends Action {
 }
 
 class LongPressAction extends Action {
-  constructor() {
+  constructor(duration = 750) {
```

Comment:
Why is the default 750? Is this what Earl Grey uses in the long press action?

Reply:
I forgot the reason, but it should be 1000 for now.

Reply:
Let's instead use the default. If no param is provided, let's call the current API, but if there is provided, let's call the new API.